src/OnlineSales/Infrastructure/CustomSqlServerMigrationsSqlGenerator.cs

```diff
@@ -192,7 +198,22 @@ private static IEntityType GetIEntityType(IModel model, string tableName)
 
     private static Type? GetType(IEntityType etype)
```

Comment:
May be this code will be better? 
`var foundType = AppDomain.CurrentDomain.GetAssemblies().Select(asm => asm.GetType(etype.Name)).FirstOrDefault(t => t is not null);`